src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.Windows.cs

```diff
@@ -539,6 +539,43 @@ internal static void CreateSymbolicLink(string path, string pathToTarget, bool i
                     Debug.Assert(!PathInternal.IsPartiallyQualified(targetPath));
                     return targetPath.ToString();
                 }
+                else if (rbSymlink.ReparseTag == Interop.Kernel32.IOReparseOptions.IO_REPARSE_TAG_APPEXECLINK)
+                {
+                    success = MemoryMarshal.TryRead(bufferSpan, out Interop.Kernel32.AppExecLinkReparseBuffer rbAppExecLink);
+                    Debug.Assert(success);
+
+                    // The target file is at index 2
+                    if (rbAppExecLink.StringCount >= 3)
```

Comment:
I did not found docs for IO_REPARSE_TAG_APPEXECLINK in time we add the code in PowerShell.

Reply:
We're starting a conversation with the Windows team about this, since the lack of docs is a blocker for us.

Reply:
PowerShell team found the const in Windows SDK so it is public and the fact opened a way to use the const.

Reply:
@iSazonov can you share a link to the public Windows SDK location where you found it?

I'm not sure such a link would qualify as "public documentation" though.

Reply:
@carlossanlop If I remember right @SteveL-MSFT found this in Windows SDK _files_ (I guess in *.cpp).

Reply:
I don't remember where exactly I got the original buffer struct from, but given the name, I think most likely from the SO post.  Since I can't find any public docs on the struct, it should probably not be relied upon as it could change.  I'll start an internal thread with the AppX team to see about getting it documented publicly.

Reply:
I found where we discuss this https://github.com/PowerShell/PowerShell/pull/10331#discussion_r316528849 - there is a link to official Microsoft example code ^-)